src/shared_utilities/drivers.jl

```diff
@@ -1511,22 +1518,44 @@ by passing `interpolation_method = Interpolations.Linear()`.
     [here](https://confluence.ecmwf.int/display/CKB/ERA5%3A+large+10m+winds).
 """
 function prescribed_forcing_era5(
-    era5_ncdata_path,
-    surface_space,
     start_date,
+    stop_date,
+    use_lowres_forcing,
+    surface_space,
     toml_dict::CP.ParamDict,
     FT;
     gustiness = 1,
     max_wind_speed = nothing,
     c_co2 = TimeVaryingInput((t) -> 4.2e-4),
     time_interpolation_method = LinearInterpolation(PeriodicCalendar()),
```

Comment:
since we use the periodic calendar by default, should this: https://github.com/CliMA/ClimaLand.jl/blob/5222662641bb93d6a61aa81c521f6c749f3a734a/src/Artifacts.jl#L46 not need the +1?